test/minikube/minikube_upgrade_helm_test.go

```diff
@@ -62,6 +63,10 @@ func upgradeAdminTest(t *testing.T, fromHelmVersion string, upgradeOptions *test
 }
 
 func upgradeDatabaseTest(t *testing.T, fromHelmVersion string, upgradeOptions *testlib.UpgradeOptions) {
+	if os.Getenv("NUODB_LICENSE") == "ENTERPRISE" {
+		t.Skip("Can not test helm upgrade in this environment")
```

Comment:
Can we include a comment or JIRA indicating what is the problem?
It's hard to track these down after some time.

Reply:
done